### PR TITLE
Internal: Use only internal test methods to update Editor controls [ED-14906]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: tests/playwright/pages/editor-page.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export default class EditorPage extends BasePage {
 		}
 
 		await this.openPageSettingsPanel();
-		await this.page.selectOption( '.elementor-control-template >> select', 'elementor_canvas' );
+		await this.setSelectControlValue( 'template', 'elementor_canvas' );
```
Comment (hein-obox) [inline tests/playwright/pages/editor-page.ts:211]: There are tests failing.

Comment (rami-elementor) [reply tests/playwright/pages/editor-page.ts:211]: Fixed

**Review by hein-obox — DISMISSED** [review]

File: tests/playwright/pages/editor-page.ts [hunk 2]
```diff
@@ -322,12 +322,20 @@ export default class EditorPage extends BasePage {
...
+		await this.page.locator( `.elementor-control-${ controlId } textarea` ).fill( value.toString() );
+	}
+
 	async setNumberControlValue( controlId: string, value: string ) {
 		await this.page.locator( `.elementor-control-${ controlId } input >> nth=0` ).fill( value.toString() );
 	}
 
 	async setChooseControlValue( controlId: string, value: string ) {
-		await this.page.locator( `.elementor-control-${ controlId } .${ value }` ).click();
+		await this.page.locator( `.elementor-control-${ controlId } input[value="${ value }"]` ).click();
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:338]: looks like selector doesn't work here. Some tests are failing

Comment (rami-elementor) [reply tests/playwright/pages/editor-page.ts:338]: Yes, I'm on it.

Comment (rami-elementor) [reply tests/playwright/pages/editor-page.ts]: Reverted. I'll fix this in a separate PR.

**Review by hein-obox — APPROVED** [review]

**Pull request merged**